### PR TITLE
installing missing dependencies for go test

### PR DESCRIPTION
**Pull request by ylebi**
fix https://github.com/scylladb/scylla-operator-images/issues/57

Installing dbus-x11 from almalinux and findutils that was missing.

### CODE REVIEW

File: base/ubi/9.4-minimal/almalinux.repo [hunk 1]
```diff
@@ -4,3 +4,10 @@ baseurl=https://repo.almalinux.org/almalinux/9/BaseOS/$basearch/os/
 gpgcheck=1
 enabled=0
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux
+
+[almalinux-appstream-9]
```
Comment (tnozicka) [inline base/ubi/9.4-minimal/almalinux.repo:8]: nit: I'd follow up the repo per file as upstreams do, like
```
ls -l /etc/yum.repos.d/
total 44
-rw-r--r--. 1 root root 1023 May  1 20:51 almalinux-appstream.repo
-rw-r--r--. 1 root root  987 May  1 20:51 almalinux-baseos.repo
-rw-r--r--. 1 root root  951 May  1 20:51 almalinux-crb.repo
-rw-r--r--. 1 root root  987 May  1 20:51 almalinux-extras.repo
-rw-r--r--. 1 root root 1107 May  1 20:51 almalinux-highavailability.repo
-rw-r--r--. 1 root root  951 May  1 20:51 almalinux-nfv.repo
-rw-r--r--. 1 root root  963 May  1 20:51 almalinux-plus.repo
-rw-r--r--. 1 root root 1107 May  1 20:51 almalinux-resilientstorage.repo
-rw-r--r--. 1 root root  939 May  1 20:51 almalinux-rt.repo
-rw-r--r--. 1 root root  951 May  1 20:51 almalinux-sap.repo
-rw-r--r--. 1 root root  999 May  1 20:51 almalinux-saphana.repo
```

File: golang/1.22/Dockerfile [hunk 2]
```diff
@@ -8,7 +8,7 @@ ENV PATH=${PATH}:${GOROOT}/bin:${GOPATH}/bin
 COPY ./download-file.sh /tmp/
 
 RUN microdnf update -y && \
-    microdnf install -y git make tar gzip gcc g++ && \
+    microdnf install -y git make tar gzip gcc g++ dbus-x11 --enablerepo=almalinux-appstream-9 && \
```
Comment (tnozicka) [inline golang/1.22/Dockerfile:11]: also `find` is missing

File: base/ubi/9.4-minimal/almalinux.repo [hunk 3]
```diff
@@ -4,3 +4,4 @@ baseurl=https://repo.almalinux.org/almalinux/9/BaseOS/$basearch/os/
 gpgcheck=1
 enabled=0
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux
+
```
Comment (tnozicka) [inline base/ubi/9.4-minimal/almalinux.repo:7]: nit: leftover

**Review by tnozicka — COMMENTED** [review]
/approve
lgtm

thanks

/assign @zimnx

**Comment by tnozicka** [comment]
(would be good to update the commit message and PR title)

File: base/ubi/9.4-minimal/almalinux-appstream.repo [hunk 4]
```diff
@@ -0,0 +1,6 @@
+[almalinux-appstream-9]
```
Comment (tnozicka) [inline base/ubi/9.4-minimal/almalinux-appstream.repo:1]: you need to update the Dockerfile as well to add it

**Review by zimnx — APPROVED** [review]
lgtm
/assign tnozicka

**Comment by zimnx** [comment]
tagging to unblock other PRs
/lgtm

**Pull request merged**